lib/dockerize_rails/docker_commands/docker_start.rb

```diff
@@ -0,0 +1,89 @@
+module DockerizeRails
+  module DockerCommands
+    module DockerStart
+      def self.start_rails
+        services = DockerizeRails::DockerCommands::Helpers.services_from_docker_compose
+        services.key?('rails') && docker_start(services['rails'], :rails)
+        0
+      rescue Docker::Error::NotFoundError => exception
+        puts
+        puts exception.to_s.red
+        puts
+        1
+      end
+
+      def self.start_mysql
+        if DRConfig.linked_database? && DRConfig.databases[DRConfig.application_env] == 'mysql'
+          services = DockerizeRails::DockerCommands::Helpers.services_from_docker_compose
+          services.key?('mysql') && docker_start(services['mysql'], :mysql)
+        end
+        0
+      rescue Docker::Error::NotFoundError => exception
+        puts
+        puts exception.to_s.red
+        puts
+        1
+      end
+
+      def self.start_postgres
+        if DRConfig.linked_database? && DRConfig.databases[DRConfig.application_env] == 'postgresql'
+          services = DockerizeRails::DockerCommands::Helpers.services_from_docker_compose
+          services.key?('postgresql') && docker_start(services['postgresql'], :postgres)
+        end
+        0
+      rescue Docker::Error::NotFoundError => exception
+        puts
+        puts exception.to_s.red
+        puts
+        1
+      end
+
+      def self.docker_start(definitions, service)
+        options = build_options(definitions, service)
+        puts options
+        container = Docker::Container.create options
+        binds =
+          if definitions.key? 'volumes'
+            { 'Binds' => definitions['volumes'].map do |volume|
+                [if volume.split(':')[0].start_with?('/')
+                  volume.split(':')[0]
+                else
+                  File.join(DockerizeRails::PATHS.current, volume.split(':')[0])
+                end,
+                volume.split(':')[1],
+                volume.length > 2 ? volume.split(':')[2] : 'rw'].join(':')
+            end }
+          end
+        puts binds
+        container.start binds
+      end
+
+      def self.build_options(definitions, service)
```

Comment:
__[rubocop]__
* Assignment Branch Condition size for build_options is too high. [31.26/20] [:ref](http://c2.com/cgi/wiki?AbcMetric)
* Method has too many lines. [24/15] [:ref](https://github.com/bbatsov/ruby-style-guide#short-methods)

<!-- Comment by [SideCI](https://sideci.com) -->